pennylane/qchem/hartree_fock.py

```diff
@@ -251,9 +254,10 @@ def _hf_energy(*args):
         """
         _, coeffs, fock_matrix, h_core, _ = scf(mol)(*args)
         e_rep = nuclear_energy(mol.nuclear_charges, mol.coordinates)(*args)
-        e_elec = anp.einsum(
+        e_elec = qml.math.einsum(
             "pq,qp", fock_matrix + h_core, mol_density_matrix(mol.n_electrons, coeffs)
         )
-        return e_elec + e_rep
+        energy = e_elec + e_rep
+        return energy.reshape(())
```

Comment:
What does `.reshape(())` do?

Reply:
To fix this jax [issue](https://github.com/google/jax/issues/2303).